### ISSUE TITLE
🐛 [Bug] Encountered bug when using  masked_fill

### ISSUE THREAD

**Issue by ttanzhiqiang** [NVIDIA/Torch-TensorRT#696]
##  Bug Description
ERROR:[TRTorch Conversion Context] - 1: [codeGenerator.cpp::createMyelinGraph::314] Error Code 1:Myelin (myelintargetsetpropertymemorysize called with invalid memory size(0).)
ERROR:[TRTorch Conversion Context] - 2:[builder.cpp::buildSerializedNetwork::417] Error Code 2:Internal Error (Assertion  enginePtr != nullptr failed)

## To Reproduce

1.
1.
1.

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - Tensorrt Version :8.0.3.4
 - Torch-TensorRT Version (e.g. 1.0.0):0.4.1
 - PyTorch Version (e.g. 1.0):1.9.1
 - CPU Architecture:
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source):pip
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version:
 - CUDA version:11.1
 - GPU models and configuration:
 - Any other relevant information:

**Comment by peri044** [comment]
@ttanzhiqiang  Can you add the steps to repro ?

**Comment by ttanzhiqiang** [comment]
To Reproduce
Steps to reproduce the behavior:
1.
class Masked_fillTest(nn.Module):
    def __init__(self):
	    super().__init__()
		
	def forward(self,x:torch.Tensor,mask:torch.Tensor)->torch.Tensor:
	    y = x.masked_fill(mask,0.0001)
		return y
	
def benchmark(trt_model,torch_model,dtype='fp32',nruns=100):
    input = torch.randn(1,4,1,1)
	mask = torch.zeros(1,1,1,1)
	
	mask = mask.cuda()
	input = input.cuda()
	torch_mask = mask.type(torch.bool)
	torch_out = torch_model(input,torch_mask)
	trt_mask = mask.type(torch.float)
	trt_out = trt_model(input,trt_mask)
	
if __name__ == "__main__":
    model = Masked_fillTest()
	model.eval().cuda()
	trt_type = torch.float
	compile_setting = {
	     "inputs":
		 trtorch.Input((10,4,1,1)),
		 trtorch.Input((10,1,1,1))
		 "enable_precision":{trt_type}
	}
	script_model = torch.jit.script(model)
	trt_ts_module = trtorch.compile(script_model,compile_setting)
	benchmark(trt_ts_module,script_model,"fp32")
2. run this

**Comment by ttanzhiqiang** [comment]
To Reproduce
Steps to reproduce the behavior:
1.
![image](https://user-images.githubusercontent.com/38750855/141057539-ba52222a-0f65-4289-ad47-2f851656d930.png)
2.run this
